docs/tasks/administer-cluster/kubelet-config-file.md

```diff
@@ -6,18 +6,20 @@ title: Set Kubelet parameters via a config file
 ---
 
 {% capture overview %}
-{% include feature-state-alpha.md %}
+{% include feature-state-beta.md %}
 
-As of Kubernetes 1.8, a subset of the Kubelet's configuration parameters may be
-set via an on-disk config file, as a substitute for command-line flags. In the
-future, most of the existing command-line flags will be deprecated in favor of
-providing parameters via a config file, which simplifies node deployment.
+A subset of the Kubelet's configuration parameters may be
+set via an on-disk config file, as a substitute for command-line flags.
+This functionality is considered beta in v1.10.
+
+Most of the existing command-line flags are deprecated in favor of providing
+parameters via a config file, which simplifies node deployment.
```

Comment:
This statement is a little bit confusing to readers if they are not aware of such 'existing command line flags'.
A different way to put it would be just state the current status. For example, "Providing parameters via a config file is the recommended approach because it simplifies node deployment and configuration management."

Reply:
done